### PR TITLE
Fix bug with scan_documents missing first result set

### PR DESCRIPTION
**Pull request by mwitek**
**Tracker Story**
https://www.pivotaltracker.com/story/show/154198499

### CODE REVIEW

File: spec/units/search_spec.rb [hunk 1]
```diff
@@ -49,12 +49,13 @@
   end
 
   let :scan_response do
-    { "_scroll_id" => "abc123", "hits" => { "total" => 2 } }
+    { "_scroll_id" => "abc123", "hits" => { "total" => 2, "hits" => [
```
Comment (davidbl) [inline spec/units/search_spec.rb]: I don't claim to understand what is happening here,  but why is `total` 2 but `hits` has only 1 element?

Comment (mwitek) [reply spec/units/search_spec.rb]: In a real use case the results would be batched, this is simulating that.

Comment (davidbl) [reply spec/units/search_spec.rb]: ah,  got it

Comment (divoxx) [reply spec/units/search_spec.rb]: Does this change to the setup reveal the issue on the test (i.e. make test fails)?

If not can you add a test that reproduce the issues?

Comment (mwitek) [reply spec/units/search_spec.rb]: I added a check to the persistence_spec here: https://github.com/doximity/es-elasticity/pull/50/files#diff-1b68ce8e83f431130ab26bbbaa97d4e7R151

Previously that was returning zero, before the change.  You think that's good enough or should I add more robust testing?

**Comment by ThomsonT** [comment]
Ready to ship.

**Pull request merged**